src/screen/SubmitTipScreen.tsx

```diff
@@ -55,6 +56,7 @@ export function SubmitTipScreen({navigation}: {navigation: NavigationProp<Dashbo
           disabled={!valid}
           onPress={() => {
             if (api && state.account) {
+              dispatch({type: 'SET_ERROR', payload: undefined});
```

Comment:
It would be better to dispatch after `start` resolves (`then` block). 
How about resetting the entire state?

Reply:
actually, I think it's unnecessary. so I removed it 7c855a3. in case the promise succeed we do navigation.goBack which will clear the state. and in case it throws we dispatch in catch block. 
also I am removing the error as soon as user changes the input in beneficiary field.